webpack/assets/javascripts/react_app/components/MemoryAllocationInput/MemoryAllocationInput.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { number, action, text } from '@theforeman/stories';
+import { number, action, text } from '@storybook/addon-knobs';
```

Comment:
```suggestion
import { number, text } from '@storybook/addon-knobs';
import { action } from '@storybook/addon-actions';
```
Looks like action needs to come from here! Once I made this change it all worked fine :+1:

Reply:
oops!  :yum: